server/pom.xml

```diff
@@ -7,7 +7,7 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.2.4.RELEASE</version>
+    <version>2.2.7.RELEASE</version>
```

Comment:
Update to 2.3.0.RELEASE is currently not possible because there are API changes with the update of elasticsearch.